Sources/Controllers/2.0/OABaseScrollableHudViewController.h

```diff
@@ -20,6 +20,10 @@ typedef NS_ENUM(NSInteger, EOADraggableMenuState)
 @property (weak, nonatomic) IBOutlet UITableView *tableView;
 @property (weak, nonatomic) IBOutlet UIView *toolBarView;
 @property (weak, nonatomic) IBOutlet UIView *scrollableView;
+@property (weak, nonatomic) IBOutlet UIView *closeButtonContainerView;
```

Comment:
Why did you extract private outlets to the base class? Base scrollable hud does not contain navBar. Please handle all extensions of the base class via inheritance and not by rewriting the base class!